.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Can we change this to only make PRs on major versions?
```suggestion
      interval: "daily"
    ignore:
      - dependency-name: "*"
        update-types: ["version-update:semver-patch", "version-update:semver-minor"]
```